estmator_project/estmator_project/views.py

```diff
@@ -0,0 +1,112 @@
+from django.views.generic import TemplateView
+from django.contrib.auth.decorators import login_required
+from django.shortcuts import render, redirect
+from django.http import HttpResponse, HttpResponseRedirect, HttpResponseNotAllowed
+from django.core.urlresolvers import reverse
+from est_quote.forms import QuoteCreateForm, ClientListForm, QuoteOptionsForm
+from est_client.models import Client, Company
+from est_client.forms import ClientCreateForm
+from est_quote.models import Quote, Category, Product, ProductProperties
+
+
+class IndexView(TemplateView):
+    template_name = 'index.html'
+
+    def get_context_data(self, **kwargs):
+        context = super(IndexView, self).get_context_data(**kwargs)
+        return context
+
+
+class QuoteView(TemplateView):
+    template_name = 'quote.html'
+
+    def get_context_data(self, **kwargs):
+        try:
+            context = super(QuoteView, self).get_context_data(**kwargs)
+            context['categories'] = Category.objects.all()
+
+            options_form = QuoteOptionsForm()
+            context['options_form'] = options_form.as_ul
+
+            context['quote_client'] = self.request.GET.get('client')
+            context['quote_name'] = self.request.GET.get('name')
+        except (KeyError, ValueError):
+            return redirect('menu')
+
+        return context
+
+
+@login_required
+def menu_view(request):
+    if request.method == 'GET':
+        client_form = ClientCreateForm()
+        quote_form = QuoteCreateForm()
+        client_list_form = ClientListForm()
+        context = {
+            'client_form': client_form.as_p,
+            'quote_form': quote_form.as_p,
+            'client_list_form': client_list_form.as_p
+        }
+        return render(
+            request, 'menu.html', context
+        )
+    else:
+        return HttpResponseNotAllowed(['GET', 'POST'])
+
+
+@login_required
+def quote_form_view(request):
+    if request.method == 'GET':
+        quote_form = QuoteCreateForm()
+        return HttpResponse(quote_form.as_p())
+    else:
+        return HttpResponseNotAllowed(['GET'])
+
+
+@login_required
+def quote_edit_form_view(request):
+    if request.method == 'GET':
+        client = Client.objects.get(id=request.GET['pk'])
+        print client
+        quotes = Quote.objects.filter(client=client)
+        print quotes
```

Comment:
be careful about leaving code intended to facilitate debugging in files when you push them to github.  it has a way of getting forgotten.